lib/Dialect/SPIRV/SPIRVOps.cpp

```diff
@@ -1944,28 +1944,24 @@ namespace {
 // Blocks from the given `spv.selection` operation must satisfy the following
 // layout:
 //
-//       +---------------------------------------------+
-//       | <header block>                              |
-//       | svp.BranchConditional %cond, ^case0, ^case1 |
-//       +---------------------------------------------+
-//         |                    |
-//         |                    v
-//         |         +-------------------------+
-//         |         | <case0>                 |
-//         |         | spv.Store %ptr %value0  |
-//         |         | spv.Branch ^merge       |---|
-//         |         +-------------------------+   |
-//         |                                       |
-//         |         +-------------------------+   |
-//         |-------->| <case1>                 |   |
-//                   | spv.Store %ptr %value1  |   |
-//                   | spv.Branch ^merge       |   |
-//                   +-------------------------+   |
-//                              |                  |
-//                              v                  |
-//                       +---------------+         |
-//                       |    <merge>    |<--------|
-//                       +---------------+
+//                   +--------------+
+//                   | header block |
```

Comment:
It's still useful to have the instructions inside these blocks as you originally did. :)